src/odometry_transformer_nodelet.cc

```diff
@@ -11,7 +11,7 @@ class OdometryTransformerNodelet : public nodelet::Nodelet {
     try {
       tf_ = std::make_shared<OdometryTransformer>(getNodeHandle(),
                                                   getPrivateNodeHandle());
-    } catch (std::runtime_error e) {
+    } catch (std::runtime_error& e) {
```

Comment:
Is it possible to pass by constant reference?
```suggestion
    } catch (const std::runtime_error& e) {
```

Reply:
done